ui/src/Home.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+function Home() {
+    return (
+        <div>
+            <h1>Home</h1>
+            <input type="button" value="Create New Poll"/>
+            <input type="button" value="View My Polls"/>
+            <input type="button" value="View All Polls"/>
+        </div>
```

Comment:
You'll want to replace the `div` here with Maggie PageLayout component; this is because we'll use that component to define stuff like the Header and make it easier to change in general. You'll probably want to do the same in the Login page JSX as well.